crates/api_common/src/person.rs

```diff
@@ -105,7 +111,7 @@ pub struct GetPersonDetailsResponse {
 
 #[derive(Debug, Serialize, Deserialize, Clone, Default)]
 pub struct GetRepliesResponse {
-  pub replies: Vec<CommentView>,
+  pub replies: Vec<CommentReplyView>,
```

Comment:
Somehow i assumed that the api would also return some kind of tree structure with this change. Is that not planned? Not that im complaining, a vec is much easier for lemmybb.

Reply:
This is a different struct, its for comment replies. 

But yes I realized after a while that its easiest to still transmit the tree flat, and have clients build the tree however they see fit. In all the UI's I've done with this tree, you pretty much have to build it anyway since updates have to search through the tree.